services/core/cards-publication/src/test/java/org/lfenergy/operatorfabric/cards/publication/services/CardProcessServiceShould.java

```diff
@@ -1078,7 +1078,7 @@ void processKeepChildCardsNull() {
     }
 
     @Test
-    void processCardWithEntitiesRequiredToRespondCorrectlySet() {
+    void processCardWithEntitiesRequiredToRespondSubsetOfEntitiesAllowedToRespond() {
```

Comment:
I don't think we should keep both tests as there is now no constraint on `entitiesRequiredToRespond`, people reading the code later might wonder why we're testing two cases (subset and not a subset). If I'm not mistaken there are no specific tests for other properties of this type, so I think that's OK if there is no test for this one.

Reply:
Done.